test/e2e/App.spec.ts

```diff
@@ -23,11 +23,10 @@ test.describe("General App Tests", () => {
 
         if (!helper.window) throw new Error("Window is not initialized");
 
-        // Aggiungi un attesa per il caricamento completo della finestra
         await helper.window.waitForLoadState("domcontentloaded");
 
         const title = await helper.window.title();
-        console.log("Window title:", title); // Log per verificare il titolo
+        console.log("Window title:", title);
```

Comment:
please do not use `console.log` unless absolutely necessary. You can use during debugging but remove it again after the debugging need is over.